### ISSUE TITLE
RTX KG2 KP didn't pass NCATS tests?

### ISSUE THREAD

**Issue by amykglen** [RTXteam/RTX#1461]
in the semantic operations call, it looks like the KG2 KP failed Chris B.'s tests (which I think are the test triples @finnagin created?)... anyone know why?

test results (KG2 is third column):

https://docs.google.com/spreadsheets/d/15IG5wG9Cb0Iq7ZAZ8ZfYfS00AzEoDs4Tr6akg2zHZcE/edit#gid=0

explanation of tests:

![Screen Shot 2021-05-13 at 8 50 07 AM](https://user-images.githubusercontent.com/49423686/118151222-3fca8480-b3c8-11eb-9e92-bf7b1b193780.png)

**Comment by finnagin** [comment]
I'm not sure. I wish there was an example of the exact query sent to the api so we could try it and see what comes back.

**Comment by amykglen** [comment]
very strange... yeah, confirmed the first triple in [the json file](https://raw.githubusercontent.com/NCATSTranslator/testing/main/onehop/test_triples/KP/Expander_Agent/RTX_KG2.json) seems to work fine on arax.ncats.io and https://arax.ncats.io/kg2/...
```
{
   "edges": {
      "e00": {
         "subject":   "n00",
         "object":    "n01",
         "predicate": "biolink:subclass_of"
      }
   },
   "nodes": {
      "n00": {
         "id":        "RO:0002630",
         "category":  "biolink:RelationshipType"
      },
      "n01": {
         "id": "biolink:negatively_regulates",
         "category":  "biolink:OntologyClass"
      }
   }
}
```
could the URL listed in the json file (in the "url" slot) be wrong? it's shown as:
```
"https://arax.ncats.io/kg2/api/rtxkg2/v1.0/ui/"
```
but if this is the exact url they're sending tests to, I think it should be this?
```
"https://arax.ncats.io/api/rtxkg2/v1.0/"
```

**Comment by finnagin** [comment]
Ah that might be the problem then. I didn't realize that was the incorrect api endpoint.

**Comment by amykglen** [comment]
yeah, I think that's the link that takes you to the Swagger documentation page, but on that Swagger page you can see the URL for requests is the latter one

**Comment by finnagin** [comment]
Oops! I'll fix that for the 2.6.3 json file.

**Comment by finnagin** [comment]
@amykglen did we ever end up standing up an endpoint for kg2.6.3?

**Comment by amykglen** [comment]
nope, we decided not to roll that one out. so no KG2.6.x endpoint at the moment.
